primitives/message-lane/src/lib.rs

```diff
@@ -134,3 +136,13 @@ impl Default for OutboundLaneData {
 		}
 	}
 }
+
+/// Returns total number of messages in the `InboundLaneData::relayers` vector.
+pub fn total_unrewarded_messages<RelayerId>(
+	relayers: &VecDeque<(MessageNonce, MessageNonce, RelayerId)>,
+) -> MessageNonce {
+	match (relayers.front(), relayers.back()) {
+		(Some((begin, _, _)), Some((_, end, _))) => end.checked_sub(*begin).and_then(|d| d.checked_add(1)).unwrap_or(0),
```

Comment:
Why do we do `d.checked_add(1)` after getting the difference?

Reply:
The range is inclusive, so if it is `[1; 1]`, then the number of messages is `(1 - 1) + 1 = 1`